clouddriver-kubernetes/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/caching/view/provider/KubernetesRawResourceProviderTest.java

```diff
@@ -42,7 +44,9 @@
 final class KubernetesRawResourceProviderTest {
 
   private KubernetesRawResourceProvider provider;
-  private KubernetesConfigurationProperties config;
+  // private KubernetesConfigurationProperties config;
```

Comment:
Nit: delete commented code (also in line 68).

Reply:
Done.

Reply:
@german-muzquiz Thank you for the feedback, I believe i've addressed it all. Please let me know if I missed something or other changes that need to be made. Thanks!